src/frontend/web_application/src/layouts/Page/components/Navigation/components/StickyNavBar/index.jsx

```diff
@@ -1,6 +1,43 @@
 import React, { Component, PropTypes } from 'react';
 import classnames from 'classnames';
 
+const throttle = (func, wait) => {
```

Comment:
I didn't need that kind of library (lodash or whatever) since es6 has almost all required method.

But yep you right, my fork has no unit tests, that's a shame. I may have saw that I can require only this function from lodash.